src/Microsoft.ML.Core/Utilities/Hashing.cs

```diff
@@ -174,6 +175,87 @@ public static uint MurmurHash(uint hash, ReadOnlySpan<char> span, bool toUpper =
             return hash;
         }
 
+        public static uint MurmurHashV2(uint hash, ReadOnlySpan<char> span, bool toUpper = false)
+        {
```

Comment:
There is a lot of context missing here. Can you please ask Ksenija for the docs she wrote and add a detailed explanation here as to why MurmurHasV2 was necessary and how it is different from the previous version?
Please add one section for the overall hashing differences and one specifically for strings.

Reply:
Added a brief description. @KsenijaS can you take a look and add anything you think is important to mention?

Reply:
Added a brief description. @KsenijaS can you take a look and add anything you think is important to mention?